src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/Editor/EditorSettingsChangedEventArgs.cs

```diff
@@ -5,9 +5,9 @@
 
 namespace Microsoft.CodeAnalysis.Razor.Editor;
 
-public sealed class EditorSettingsChangedEventArgs : EventArgs
```

Comment:
Not sure we want to keep this in a separate file or move next to the advanced equivalent.